src/aiida/tools/dumping/processes.py

```diff
@@ -0,0 +1,454 @@
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Functionality for dumping of WorkChains and CalcJobs."""
+
+from __future__ import annotations
+
+import logging
+from pathlib import Path
+from types import SimpleNamespace
+from typing import Any, List, Optional
+
+import yaml
+
+from aiida.common import LinkType
+from aiida.common.exceptions import NotExistentAttributeError
+from aiida.orm import (
+    CalcFunctionNode,
+    CalcJobNode,
+    CalculationNode,
+    LinkManager,
+    ProcessNode,
+    WorkChainNode,
+    WorkflowNode,
+    WorkFunctionNode,
+)
+from aiida.orm.utils import LinkTriple
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ProcessDumper:
+    def __init__(
+        self,
+        include_inputs: bool = True,
+        include_outputs: bool = False,
+        include_attributes: bool = True,
+        include_extras: bool = True,
+        overwrite: bool = True,
```

Comment:
Should the default perhaps be `False`, just as the CLI option?

Reply:
Yes, good catch!